src/sentry/static/sentry/app/views/eventsV2/results.tsx

```diff
@@ -99,9 +106,70 @@ class Results extends React.Component<Props, State> {
     }
   }
 
-  async fetchTotalCount() {
+  canLoadEvents = async () => {
```

Comment:
`canLoadEvents` sounds like is just verifying whether events can be loaded, but it's actually fetching them? -> `loadEvents`?